EIPS/eip-2612.md

```diff
@@ -4,38 +4,39 @@ title: permit – 712-signed approvals
 
-This ERC extends the ERC-20 standard with a new function `permit`, which allows users to modify the `allowance` mapping using a signed message, instead of through `msg.sender`. 
+This ERC extends the EIP-20 standard with a new function `permit`, which allows users to modify the `allowance` mapping using a signed message, instead of through `msg.sender`. 
 
-For an improved user experience, the signed data is structured following [ERC-712](./eip-712.md), which already has wide spread adoption in major RPC providers.
+For an improved user experience, the signed data is structured following [EIP-712](./eip-712.md), which already has wide spread adoption in major RPC providers.
 
-**_Note:_** ERC-20 must be performed by an EOA unless the address owning the token is actually a contract wallet. Although contract wallets solves many of the same problems that motivates this EIP, they are currently only scarcely adopted in the ecosystem. Contract wallets suffer from a UX problem -- since they separate the EOA `owner` of the contract wallet from the contract wallet itself (which is meant to carry out actions on the `owner`s behalf and holds all of their funds), user interfaces need to be specifically designed to support them. The `permit` pattern reaps many of the same benefits while requiring little to no change in user interfaces.
+**_Note:_** EIP-20 must be performed by an EOA unless the address owning the token is actually a contract wallet. Although contract wallets solves many of the same problems that motivates this EIP, they are currently only scarcely adopted in the ecosystem. Contract wallets suffer from a UX problem -- since they separate the EOA `owner` of the contract wallet from the contract wallet itself (which is meant to carry out actions on the `owner`s behalf and holds all of their funds), user interfaces need to be specifically designed to support them. The `permit` pattern reaps many of the same benefits while requiring little to no change in user interfaces.
 
 ## Motivation
-While ERC-20 tokens have become ubiquitous in the Ethereum ecosystem, their status remains that of second class tokens from the perspective of the protocol. The ability for users to interact with Ethereum without holding any ETH has been a [long outstanding goal](./eip-101.md) and the [subject](./eip-1077.md) [of](./eip-777.md) [many](https://github.com/ethereum/EIPs/issues/1776) [EIPs](./eip-1271.md).
+While EIP-20 tokens have become ubiquitous in the Ethereum ecosystem, their status remains that of second class tokens from the perspective of the protocol. The ability for users to interact with Ethereum without holding any ETH has been a [long outstanding goal](./eip-101.md) and the [subject](./eip-1077.md) [of](./eip-777.md) [many](https://github.com/ethereum/EIPs/issues/1776) [EIPs](./eip-1271.md).
```

Comment:
```suggestion
While EIP-20 tokens have become ubiquitous in the Ethereum ecosystem, their status remains that of second class tokens from the perspective of the protocol. The ability for users to interact with Ethereum without holding any ETH has been a long outstanding goal and the subject of many EIPs.
```
Required: no external links.
Recommended to a point that it might as well be required: don't needlessly reference other EIPs

Reply:
I commented on this above https://github.com/ethereum/EIPs/pull/5506#issuecomment-1220846491:
>The validator is complaining about references to non-Final EIPs. I'd like to challenge this rule. Referring to prior art should be encouraged even if that prior art is a non-Final EIP. As far as I can tell the only problem would be to extend or somehow depend on a non-Final EIP. This also touches on the ambiguity of the `requires:` field which I've been told should be intepreted as "references:" rather than as dependencies. Dependencies should only be allowed to EIPs in later stages, but references should not care.

Reply:
I am aware. We are not going to provide exemptions for these particular references, as they are by no means required for the general understanding of the EIP.

Reply:
I'm not interested in discussing this case too much and I will gladly remove the references.

But I do think that referencing prior art is important, among other things to avoid constantly reinventing the wheel with things that have been tried before, and would consider it valuable to have references in an EIP to the attempted EIPs that preceded it. The current requirement seem to not leave room for this.

Reply:
> The current requirements seem to not leave room for this.

This is something we are currently discussing. I think I have a better idea of how I can actually do this in a non-subjective manner, but that will have to be a future modification to EIP-1.